Util/pose.py

```diff
@@ -67,6 +67,9 @@ def to_array(self) -> np.ndarray:
     def to_dict(self) -> Dict[str, float]:
         return {'x': self.x, 'y': self.y, 'orientation': self.orientation}
 
+    def flip_x(self):
+        return Pose.from_values(-self.x, self.y, (2 * np.pi - self.orientation) % (2 * m.pi) - m.pi)
```

Comment:
make a wrap_to_pi helper function? avoid the confusion of seeing this line

Reply:
maybe mirror_x is a better name? not sure. flip_x seems to just flip the x component and hide the orientation. #veryneatpickyforgiveme

Reply:
The problem is that we already have a wrap_to_pi in geometry.py. But we can not import it since geometry depends on pose

Reply:
And we can not put it inside pose since it's also confusing.

Reply:
mirror_x fait bin plus de sense

Reply:
actually, it's the `import from` that cause problem. module import `import Pose` should avoid the circular import.